pyproject.toml

```diff
@@ -69,6 +69,7 @@ addopts = "--cov=. --cov-report=xml"
 omit = [
     "*/tests/*",
     "venv/*",
+    "alns/weights/*",

```

Comment:
Instead of this, can we have one test that checks importing `alns.weights` warns?

E.g.

```python
def test_alns_weights_warns_deprecation():
    with assert_warns(DeprecationWarning):
        import alns.weights   # noqa
```